FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Extensions/ConfigurationRegistrationExtensions.cs

```diff
@@ -63,6 +67,21 @@ private static void ValidateConfiguration(this IServiceCollection services)
                 throw new ConfigurationErrorException($"Target azure container name '{jobConfiguration.ContainerName}' can not be empty.");
             }
 
+            FilterConfiguration filterConfiguration;
```

Comment:
Just curious why the filter configuration needs the `try` term while other configurations needn't?

Reply:
filter configuration contains enum type field `FilterScope`, it will throw an exception if users give an invalid enum string without `try`.
Thanks for the comments, I will add try for other configurations containing Enum type.